types/react-draft-wysiwyg/test/render-with-test-id.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { Editor } from 'react-draft-wysiwyg';
+
+const RenderWithTestId = () => (
+    <div>
+        <Editor webDriverTestID="test" />
```

Comment:
Hi @imechZhangLY, **webDriverTestId** prop renders **data-testid** attribute to the draft editor element. The **data-testid** attribute is commonly used as a selector in Unit Testing or End-to-end Testing frameworks.

This prop is not documented in official document, however it could be found in the implementation docs [here](https://npmdoc.github.io/node-npmdoc-draft-js/build/apidoc.html#apidoc.element.draft-js.Editor.prototype.render). To also further confirm its support by draft-js [type def](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0b5bfba2994c91a099cd5bcfd984f6c4c39228e5/types/draft-js/index.d.ts#L169) and a discussion in [cypress-io repo](https://github.com/cypress-io/cypress/issues/596#issuecomment-370773333)

Hope this clarifies your question! Thanks